scout/server/blueprints/variants/views.py

```diff
@@ -508,6 +487,7 @@ def cancer_sv_variants(institute_id, case_name):
         show_dismiss_block=controllers.get_show_dismiss_block(),
         total_variants=variants_stats.get(variant_type, {}).get(category, "NA"),
         variant_type=variant_type,
+        expand_search=controllers.get_expand_search(request.form),
```

Comment:
Wasn't alphabetical order kind of nice? 😊

Reply:
I didn't notice, I'll fix it!

Reply:
Hihi, not all of them are, as evident from this. But thanks! 😁